### ISSUE TITLE
[API] Specify artist as artist

### ISSUE THREAD

**Issue by pagdot** [skaupper/VirtualJukebox#22]
The current API documentation has a field for tracks labeled "author" which contains the artist of the track. I recommend using the label "artist" instead.

Current state:

```
    "currently_playing": {
        "track_id": "<track_id>",
        "title": "<track_title>",
        "album": "<album_name>",
        "author": "<author_name>",
        "duration": "<duration>",
        "icon_uri": "<uri>",
        "playing": true|false,
        "playing_for": <playing_for_ms>
    },
```

recommended:

```
    "currently_playing": {
        "track_id": "<track_id>",
        "title": "<track_title>",
        "album": "<album_name>",
        "artist": "<artist_name>",
        "duration": "<duration>",
        "icon_uri": "<uri>",
        "playing": true|false,
        "playing_for": <playing_for_ms>
    },
```

**Comment by skaupper** [comment]
addressed in #16

**Comment by skaupper** [comment]
I close the issue as soon as the PR is merged
